components/sup/src/util/users.rs

```diff
@@ -33,6 +33,10 @@ fn check_pkg_user_and_group(pkg_install: &PackageInstall) -> Result<Option<(Stri
     let svc_group = try!(pkg_install.svc_group());
     match (svc_user, svc_group) {
         (Some(user), Some(group)) => {
+            // a package has a SVC_USER and SVC_GROUP defined,
+            // these MUST exist in order to continue
+            debug!("SVC_USER = {}", &user);
```

Comment:
Let's drop these debug statements. I've been cleaning out debug/warn/error output from re-usable functions. Logging works best when it's within the main program flow to help tell the story to the person reading the log. When it's tucked away in a function we might call pretty often it becomes a bit less useful ;)